runtime/doc/editing.txt

```diff
@@ -801,8 +801,9 @@ current window.  The two windows then share this list, until one of them uses
 USING THE ARGUMENT LIST
 
 						*:argdo*
-:argdo[!] {cmd}		Execute {cmd} for each file in the argument list.
-			It works like doing this: >
+:[range]argdo[!] {cmd}	Execute {cmd} for each file in the argument list or
+                        if [range] is specified only for arguments in that
+			range. It works like doing this: >
```

Comment:
Mixed tabs and spaces. Double space after sentence punctuation.

I'll add a note about whitespace/formatting at the bottom.